Description: Use the system pcre instead of something in third-party
 There is a version of pcre in third-party that shouldn't be used.
Author: Giuseppe Lavagetto <glavagetto@wikimedia.org>
Forwarded: no
Last-Update: 2014-10-21

Index: hhvm/third-party/CMakeLists.txt
===================================================================
--- hhvm.orig/third-party/CMakeLists.txt
+++ hhvm/third-party/CMakeLists.txt
@@ -24,7 +24,6 @@
 set(THIRD_PARTY_HEADERS)
 
 list(APPEND THIRD_PARTY_MODULES
-  pcre
   fastlz
   libafdt
   libmbfl
